server.js

```diff
@@ -6038,6 +6039,8 @@ function versionColor(version) {
   var first = version[0];
   if (first === 'v') {
     first = version[1];
+  } else if (/^version\/[0-9]/.test(version)) {
+    version = 'v' + version.substring(8);
```

Comment:
Could you separate improvements unrelated to the stated goal of the PR in a different PR, please?